### PR TITLE
Update models.go

### PR DESCRIPTION
**Pull request by pp010**
I missed it in last PR. Can we merge this and create a tag!?

Thanks for your contribution!
Hi, if there is an issue, that your PR adresses, please link it!

### CODE REVIEW

File: models.go [hunk 1]
```diff
@@ -496,7 +496,7 @@ type RealmRepresentation struct {
 	UserFederationMappers               []interface{}     `json:"userFederationMappers,omitempty"`
 	UserFederationProviders             []interface{}     `json:"userFederationProviders,omitempty"`
 	UserManagedAccessAllowed            bool              `json:"userManagedAccessAllowed"`
-	Users                               []interface{}     `json:"users,omitempty"`
+	Users                               []User            `json:"users,omitempty"`
```
Comment (pp010) [inline models.go]: I am not sure why `[]interface{}` does not take `[]User `. It should be like a wild card and take any array input.

**Comment by pp010** [comment]
I missed it in last PR. Can we merge this and create a tag!?

**Comment by Nerzal** [comment]
done

**Comment by SVilgelm** [comment]
https://github.com/Nerzal/gocloak/releases/tag/v3.1.0

**Comment by pp010** [comment]
Thanks :)

**Pull request merged**